test/interface/jacobian.jl

```diff
@@ -15,13 +15,6 @@ using Test
         nothing
     end
 
-    nWfacts = Ref(0)
-    function Wfact(W, u, h, p, dtgamma, t)
```

Comment:
Why does this PR remove the Wfact tests? We could skip them for Rosenbrock23 if needed but for TRBDF2 they should still work, shouldn't they?

Reply:
Wfact only gets used when transform=false and rosenbrock23 was the last solver that didn't w-transform.

Reply:
once I merge both of the OrdinaryDiffEq prs, I'm going to open a PR that deprecates or removes W_fact from ODEFuncton